index.html

```diff
@@ -14,17 +14,15 @@
             <p class="text-gray-400 text-sm">Sustainable Education Foundation</p>
         </div>
         <div class="flex text-xs font-light space-x-5">
-            <a class="inline-block" href="https://github.com/sef-global" target="_blank">Github</a>
-            <a class="inline-block" 
+            <a class="inline-block px-1 py-2" href="https://github.com/sef-global" target="_blank">Github</a>
+            <a class="inline-block px-1 py-2" 
                href="https://join.slack.com/t/sefheadquarters/shared_invite/zt-9w6v9rdu-uJGLrMKLTNFzQEVPC4sbOw" 
                target="_blank">
                  Slack
             </a>
-            <a class="inline-block" href="https://sef.discourse.group" target="_blank">SEF Hive</a>
-            <a href="https://github.com/sef-global/sef-site/blob/master/CONTRIBUTING.md" target="_blank">
-                <span class="bg-blue-600 rounded-full p-1 px-2">
-                    Best Practices
-                </span>
+            <a class="inline-block px-1 py-2" href="https://sef.discourse.group" target="_blank">SEF Hive</a>
+            <a href="#_" class="inline-flex items-center justify-center font-medium transition-all px-4 py-2 text-center no-underline align-middle transition-all duration-300 ease-in-out bg-blue-600 border-solid rounded-full cursor-pointer select-none hover:text-white hover:border-white focus:shadow-xs focus:no-underline">
+                Best Practices
```

Comment:
Let's break this line down, it's hard to read.

Reply:
requested changes made, please take a loot.